src/Sticky.jsx

```diff
@@ -276,6 +276,12 @@ class Sticky extends Component {
         this.update();
     }
 
+    componentDidUpdate(prevProps, prevState) {
+        if (prevState.status !== this.state.status) {
+            this.props.onStickyStateChange(this.state.status === 2);
```

Comment:
Some minor issues here:
1. Should use `STATUS_FIXED` instead of comparing with `2`.
2. I prefer not calling then calling an empty function for the performance reason.

This idea is good, but I am thinking of more status: `ORIGINAL`, `STICK_TO_TOP`, `STICK_TO_DOWN`, and `RELEASE`. I mean for the flexibility, we can return an object like
```js
{
   status: 'original' // and 'fixed' for now, I also swing between number or string.
}
```
How do you think, @eligolding?